homeassistant/helpers/condition.py

```diff
@@ -167,6 +167,7 @@ def async_numeric_state(hass: HomeAssistant, entity, below=None, above=None,
         value = float(value)
     except ValueError:
         _LOGGER.warning("Value cannot be processed as a number: %s", value)
+        _LOGGER.warning("Offending entity: %s .", entity)
```

Comment:
Why not put that in one warning?

Reply:
I'm not a coder (i did search and for instance found https://realpython.com/python-f-strings/ ) and had trouble getting the syntax right with 2 variables. Sorry.